BUILDING.md

```diff
@@ -0,0 +1,136 @@
+# How to build MetalK8s
+
+Our build system is based on [doit](http://pydoit.org/).
+
+## Dependencies
+
+- [Python3.6+](https://www.python.org/)
+
+## Setup
+
+```shell
+python3 -m venv buildenv
```

Comment:
Can we set up a `tox` target for this, which managed all of this? So we can run `tox -e build` (~ `doit`), `tox -e build -- list` (~ `doit list`) etc?

Reply:
Solved with a wrapper script so that the users don't have to deal with the venv themselves.